style.css

```diff
@@ -627,6 +622,113 @@ form textarea {
   background-color: transparent;
 }
 
+/* Creat style for popup window */
+
+.modal-section {
+  position: fixed;
+  width: 100%;
+  min-height: 100%;
+  transition: 0.2s ease-in-out;
+  background-color: rgba(165, 185, 224, 0.5);
+  -webkit-backdrop-filter: blur(5px);
+  margin-bottom: 50px;
+  backdrop-filter: blur(5px);
+  display: none;
+  overflow: auto;
+}
+
+.nav-toggle {
+  display: flex;
+  flex-direction: column;
+}
+
+.card-modal {
+  position: absolute;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 24px;
+  margin: 4%;
+}
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, please make sure you follow the design guidelines as much as possible. 
- [ ] You may fix the responsiveness of your popup so that your design can replicate the given design for both mobile and desktop views.
- [ ] Kindly make sure the image is a replica of the design given. Also, remove the horizontal scroll from your design so the popup can be fully visible on one page without using the vertical bar. 
- Your design
![image-scroll](https://user-images.githubusercontent.com/63917725/225054388-a1d087b5-a2d3-4a43-80d8-15989c0ea6db.PNG)

- [ ] You may also fix the text of your popup as well as the buttons so that they replicate what is given in the design.
- Your design
![button and text](https://user-images.githubusercontent.com/63917725/225054723-91feaee0-ef2d-48b3-8d5d-717f97013c01.PNG)
- Given design.
![given-design](https://user-images.githubusercontent.com/63917725/225055003-0e8a19c9-29cf-46bf-88af-50bd11c4785d.PNG)

Reply:
all solved, thanks.